docs/windows.md

```diff
@@ -170,12 +170,12 @@ depending on whether you are using your own [signed](https://docs.microsoft.com/
 OVS kernel driver or you want to use the test-signed driver provided by Antrea,
 you will need to invoke the `Install-OVS.ps1` script differently (or not at all).
 
-| Containerized OVS daemons? | Test-signed OVS driver? | Run this command |
-| -------------------------- | ----------------------- | ---------------- |
-| Yes                        | Yes                     | `.\Install-OVS.ps1 -InstallUserspace $false` |
-| Yes                        | No                      | N/A |
-| No                         | Yes                     | `.\Install-OVS.ps1` |
-| No                         | No                      | `.\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage>` |
```

Comment:
-ImportCertificate flag is removed in Install-OVS.ps1
does it mean certificate import is not required for test signed driver !

Reply:
No, the certificate is import no matter it is for test-signed driver or a signed driver in the latest change.

Reply:
I think `N/A` might be confusing, does it mean we don't support this feature or no specific command is required to use it? Maybe @antoninbas  can offer more suggestions in doc change?

Reply:
I am fine with "Not needed" or "Not required" if your prefer

Reply:
updated.